packages/functions/src/auth.ts

```diff
@@ -85,12 +26,21 @@ export const queryAuthStatus2 = functions
   .region(__functionsZone__)
   .https.onCall(
     wrapHttpsOnCallHandler(
-      "queryAuthStatus2",
-      async (payload: QueryAuthStatusPayload): Promise<AuthStatus> => {
-        // validate payload
-        checkRequiredFields(payload, ["organization", "authString"]);
-
-        const { organization, authString } = payload;
+      "queryAuthStatus",
+      async (
+        payload: QueryAuthStatusPayload,
+        { auth }
+      ): Promise<AuthStatus> => {
+        // Organization is required to even start the functionality
+        checkRequiredFields(payload, ["organization"]);
+        const { organization } = payload;
+
+        await checkUser(organization, auth);
```

Comment:
`checkUser` has no side effects AFAICT.
This invocation can probably be removed.